CatAnalyzer/plugins/ttbbLepJetsAnalyzer.cc

```diff
@@ -175,73 +222,94 @@ class ttbbLepJetsAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResource
 // constructors and destructor
 //
 ttbbLepJetsAnalyzer::ttbbLepJetsAnalyzer(const edm::ParameterSet& iConfig):
-  isMC_    (iConfig.getUntrackedParameter<bool>("sampleLabel",  true)),
```

Comment:
it would be better to remove the data and MC dependency in configuration level 
please use 
bool isData = iEvent.isRealData();